gradle.properties

```diff
@@ -26,7 +26,6 @@ systemProp.dependency.analysis.test.analysis=false
 
 # List of project that still have dependency analysis warnings
 unmigratedProjects=\
-  build-cache-base,build-cache-packaging,hashing,snapshots,\
```

Comment:
💭 I took the opportunity to re-enable these checks as part of the PR, too. It doesn't have a big effect, and this way we can be sure that we only have the dependencies we need in the build-cache library projects.